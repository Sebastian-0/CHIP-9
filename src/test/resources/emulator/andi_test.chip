
ldi A, 0xFF

# Negative flag
andi 0x8F
sout A
sout FLAGS

# No flag
andi 0x7C
sout A
sout FLAGS

# Zero flag
andi 0x0
sout A
sout FLAGS

hcf