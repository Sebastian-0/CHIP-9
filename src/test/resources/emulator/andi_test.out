
0x8F
64

0x0C
0

0x00
128